01_02_Combat_Rules.txt

```diff
@@ -383,20 +385,32 @@ Performing a Skill Check (other than Getting Bearings): 1 Action
 Conversing With Partners: Free Action
 Getting Your Bearings / Non Skill-Based Perception: Free Action
 
+Free Actions:
+	Free Actions refer to tasks that take miniscule amounts of time, even
+in pitched combat, like looking around, speaking briefly, or reflexively 
+shutting your eyes after being treated to a flash-bang. Aside from your 
+GM's patience and willingness to believe your character can do everything you 
+want in a negligible timescale, there is no limit to the number of Free Actions 
+a character may take, and they may be used at any time.
+
 ===== Reactions & Held Actions =====
 
-	Certain abilities or circumstances will allow you to take a special action 
-in response to a triggering event. A reaction occurs at the immediate moment 
-that it is triggered, whether it is your turn or not. The most common reaction 
-you will encounter will be from Suppression (described in "Ranged Combat", 
-below) or Opportunity Attacks (described in "Melee Combat", above). After the 
-reaction occurs, play continues as normal (if it is an opponents turn, they 
-continue their turn). 
+Reactions:
+	Certain abilities or circumstances will allow you do something 
+in response to a triggering event. Doing so requires using a special kind 
+of Action called a Reaction. Reactions occur at the immediate moment 
+that they are triggered, regardless of whose turn it is. Common uses of 
+Reactions include Opportunity Attacks (described in "Melee Combat", above) 
+and Held Actions, below. After the Reaction occurs, play continues as normal. 
```

Comment:
Do we really want held actions to be a type of Reaction? I'm wary of discouraging held actions like that, and I like the tactical depth allowed when you can both take a reaction (based on a feat, ability, opportunity attack) and shift part of your turn within the initiative order.

Reply:
While this problem was technically created by these changes, the actual problem text isn't something I touched. I'm not sure where I fall on this question (therefore it is a good question 🐢), but since I've scope crept so much already would you be comfortable classifying this as New Issue instead of Blocker?

Reply:
Aye aye!

Reply:
Now that the Reaction cap has moved to 3, has this become a nonissue? @rosebrallier